CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 - Updated minimum version of Terraform to avoid a bug (by @dpiddockcmp)
 - Fix cluster_oidc_issuer_url output from list to string (by @chewvader)
 - Fix idempotency issues for node groups with no remote_access configuration (by @jeffmhastings)
+- Improve UX of create new EKS cluster (by @ryanooi)
```

Comment:
Can you update this to describe your changes more accurately? i.e. "changed logic around SG and SG rule creation"

Reply:
Updated. Please help review again :)